doc/classes/ProjectSettings.xml

```diff
@@ -1436,6 +1436,10 @@
 		<member name="internationalization/rendering/force_right_to_left_layout_direction" type="bool" setter="" getter="" default="false">
 			Force layout direction and text writing direction to RTL for all controls.
 		</member>
+		<member name="internationalization/rendering/root_node_auto_translate" type="bool" setter="" getter="" default="true">
```

Comment:
A "rendering" subsection for this is a bit weird, but it's already what's used for RTL so I agree it's fine to reuse it. Naming things is so hard :D